tests/src/system/basic/ActionTests.scala

```diff
@@ -78,4 +88,25 @@ class ActionTests
                     """[OK] message terminated successfully""")
             }
     }
+
+    /**
+     * Test an action that does not exceed the allowed execution timeout of an action.
+     */
+    it should "fail on creating an action with exec which is too big" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "TestActionCausingParamsTooBig"
+
+            val actionCode = new File(s"$testActionsDir${File.separator}$name.js")
+            actionCode.createNewFile()
+            val pw = new PrintWriter(actionCode)
+            pw.write("a" * ((actionCodeLimit.toBytes / 2) + 1).toInt)
+            pw.close
+
+            assetHelper.withCleaner(wsk.action, name, confirmDelete = false) {
```

Comment:
Since the REQUEST_ENTITY_TOO_LARGE exit code and the Go CLI exit code both are based off an HTTP failing status code, no Go CLI changes are needed.  The exit code for HTTP failures is statuscode - 256.  In this case the exit code will be 413 - 256 = 157.